docs/bis/leftNav.md

```diff
@@ -1,87 +1,131 @@
+  - [Categories](./guide/fundamentals/categories.md)
+
+- #### Data Organization
+
+  - [Information Hierarchy](./guide/data-organization/information-hierarchy.md)
+  - [Modeling Perspectives](./guide/data-organization/modeling-perspectives.md)
+  - [Top of the World](./guide/data-organization/top-of-the-world.md)
+  - [Single Responsible-Party Principle](./guide/data-organization/srpp.md)
+  - [Organizing Models and Elements](./guide/data-organization/organizing-models-and-elements.md)
+  - [Spatial Composition](./guide/data-organization/spatial-composition.md)
+  - [Modeling Systems](./guide/data-organization/modeling-systems.md)
+  - [Organizing Definition Elements](./guide/data-organization/organizing-definition-elements.md)
+
+- #### Physical Modeling Perspective
+
+  - [3D Guidance](./guide/physical-perspective/3d-guidance.md)
+  - [PhysicalModel Hierarchy](./guide/physical-perspective/physical-hierarchy-organization.md)
+  - [Physical Models and Elements](./guide/physical-perspective/physical-models-and-elements.md)
+  - [Physical Materials](./guide/physical-perspective/physical-materials.md)
+  - [Quantity Takeoffs: Guidelines](./guide/physical-perspective/qto-guidelines.md)
+
+- #### Other Modeling Perspectives
+
+  - [Functional Models and Elements](./guide/other-perspectives/functional-models-and-elements.md)
+  - [Analysis Models and Elements](./guide/other-perspectives/analysis-models-and-elements.md)
+  - [Information Models and Elements](./guide/other-perspectives/information-models-and-elements.md)
+
+- #### Schema Evolution
+
+  - [Schema Customization](./guide/schema-evolution/schema-customization.md)
+  - [Data Evolution Across Time](./guide/schema-evolution/data-evolution-across-time.md)
+  - [Schema Versioning](./guide/schema-evolution/schema-versioning-and-generations.md)
+  - [Schema Production Status](./guide/schema-evolution/schema-production-status.md)
+  - [Schema Design Guidelines](./guide/schema-evolution/schema-design-guidelines.md)
+
+- #### BIS Naming Guidelines
+
+  - [Rules and Recommendations](./guide/naming-guidelines/rules-and-recommendations.md)
+  - [Special Terms](./guide/naming-guidelines/special-terms.md)
+  - [Summary of Exceptions](./guide/naming-guidelines/summary-of-exceptions.md)
+  - [Abbreviations and Acronyms](./guide/naming-guidelines/standard-abbreviations-and-acronyms.md)
+  - [Relationship “Strengths”](./guide/naming-guidelines/standard-relationship-strengths-names.md)
+
+- #### Other topics
+
+  - [BIS Schema Units](./guide/other-topics/units.md)
+  - [BIS Schema KindOfQuantities](./guide/other-topics/kindOfQuantities.md)
+  - [BIS Schema Validation](./guide/other-topics/bis-schema-validation.md)
+
+- #### References
+
+  - [BIS Glossary](./guide/references/glossary.md)
+  - [Class-diagram Conventions](./guide/references/class-diagram-conventions.md)
+  - [Instance-diagram Conventions](./guide/references/instance-diagram-conventions.md)
 
 ### Domain Schemas
```

Comment:
@ColinKerr @diegoalexdiaz can you guys review this section?

Reply:
It looks to me that this leftNav is the result of upgrading the one in production according to the newer BeMetalSmith code. There is [another PR](https://github.com/iTwin/itwinjs-core/pull/4893) with several improvements to the BIS docs that also contains changes to this leftNav, including upgrades based on the newer BeMetalSmith.

Reply:
Right. Do you want to get that PR through so I can merge it in here?

Reply:
The other PR is taking longer to get reviewed than I originally expected, so we better get your PR merged first.